scio-parquet/src/test/scala/com/spotify/scio/parquet/avro/ParquetAvroIOTest.scala

```diff
@@ -119,6 +121,63 @@ class ParquetAvroIOTest extends ScioIOSpec with TapSpec with BeforeAndAfterAll {
     ()
   }
 
+  it should "write and read SpecificRecords with logical types" in {
+    val records =
+      (1 to 10).map(_ =>
+        TestLogicalTypes
+          .newBuilder()
+          .setTimestamp(DateTime.now())
+          .setDecimal(BigDecimal.decimal(1.0).setScale(2).bigDecimal)
+          .build()
+      )
+    val path = dir.toPath.resolve("logicalTypesSr").toString
+
+    val sc1 = ScioContext()
+    sc1
+      .parallelize(records)
+      .saveAsParquetAvroFile(path)
+    sc1.run()
+    ()
+
+    val sc2 = ScioContext()
+    sc2
+      .parquetAvroFile[TestLogicalTypes](s"$path/*.parquet")
+      .map(identity) should containInAnyOrder(records)
+
+    sc2.run()
+    ()
+  }
+
+  it should "write and read GenericRecords with logical types" in {
```

Comment:
do you think it's useful to do a cross test e.g. write with Generic and read with SpecificRcord?

Reply:
I think in that regard, that's more in the realm of native Avro functionality than any Scio functionality that we own. When I was validating this change with real data, I did do a run with GenericRecord writes and SpecificRecord reads, which succeeded :) but I don't think we need a devoted unit test for it.